administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -188,10 +188,13 @@ COM_LOCALISE_CANNOT_REMOVE_DEFAULT_LANGUAGE="Localise cannot remove default lang
 
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT="Translate"
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT_DESC="Translate description"
+COM_LOCALISE_CHECKBOX_TRANSLATION_GRAMMAR_CASE="Grammar case"
 COM_LOCALISE_CONFIRM_TRANSLATION_SAVE="Before saving, make sure you have set if the translation is 'Complete' or not in the 'Details' tab."
+COM_LOCALISE_CONFIRM_TRANSLATION_SAVE_NOTINREF="Before saving, make sure to review if within the 'Not in ref' section you have selected the corresponding checkboxes of the strings to be deleted."
+COM_LOCALISE_DELETE="Delete"
 COM_LOCALISE_ERROR_TRANSLATION_FILESAVE="Error saving translation file %s"
 COM_LOCALISE_FIELDSET_TRANSLATION_DETAIL="Details"
-COM_LOCALISE_FIELDSET_TRANSLATION_DETAIL_DESC="Values ​​of inactive fields are set in general settings of Localise component."
+COM_LOCALISE_FIELDSET_TRANSLATION_DETAIL_DESC="Values of inactive fields are set in general settings of Localise component."
```

Comment:
Suggest
`COM_LOCALISE_FIELDSET_TRANSLATION_DETAIL_DESC="Values of inactive fields are set in the component options"

Reply:
I will apply the suggestion but no string changed from my part there. It was due from the Github edition of the file it was showing a "red dot" there, between the words, so.. odd dot.. i delete  the dot XD.